esp/src/src/Graph7Widget.ts

```diff
@@ -185,7 +186,7 @@ export class Graph7Widget {
 
         this.initGraph();
 
-        this.doInit(params);
+        this.doInit(params.Wuid);
```

Comment:
This may break other places it is used?

Reply:
I wasn't sure about this also, but wasn't sure what over places it might be called. The graph widget in the dojo version of playground seems to still behave normally. The changes to DojoAdapter should ensure Wuid is always set on params I thought.

I did have to loosen the type on params in DojoAdapter (around line 17) from "object" to "any" to avoid a compiler issue or two. Should I have instead created an interface for something like below?

interface WidgetParams {
    Wuid?: string;
    Sequence?: number;
}

export interface DojoAdapterProps {
    widgetClassID?: string;
    widgetClass?: any;
    params?: WidgetParams; // instead of object or any
    delayProps?: object;
    onWidgetMount?: (widget) => void;
}

Reply:
If you open a "Legacy Graph" from the Graphs tab on the WU Details Page.

In general specific interfaces are preferable to an `any`, but you example above looks a little specific to WU related "Widgets"

Reply:
The Legacy graph looks to be ok to me